Squablr/View Controllers/UserFeedViewController.m

```diff
@@ -8,17 +8,21 @@
 #import "UserFeedViewController.h"
 
 @interface UserFeedViewController ()
+@property (nonatomic, strong) NSMutableArray *currentUserProfileInfo;
 @property (nonatomic, strong) NSMutableArray *arrayOfUserObjects;
-@property (nonatomic, strong) NSMutableArray *userProfilePhotos;
+@property (nonatomic, strong) NSMutableArray *userPlaceHolderArray;
```

Comment:
Why do we have all these new properties? Guessing it's leftover from what you were trying to do previously?

Reply:
Yes, they are